Translations/fr.po

```diff
@@ -8924,24 +8927,25 @@ msgid ""
 "said that when this holy armor is again needed, a hero will arise to don "
 "Valor once more. Perhaps you are that hero..."
 msgstr ""
-"La Porte du Sang et les Salles du Feu sont des repères d'origine mystique. "
-"Où que se trouve ce livre que vous lisez, c'est sûrement un lieu d'une "
+"Le Portail du Sang et les Salles du Feu sont des repères d'origine mystique. "
+"Où que se trouve ce livre que vous avez lu, c'est certainement un site d'une "
 "grande puissance.\n"
 " \n"
-"Les légendes parlent d'un piédestal qui est sculpté dans de la pierre "
-"d'obsidienne et a une mare de sang bouillant au sommet de sa surface "
-"incrustée d'os. Il y a aussi des allusions aux Pierres de Sang qui ouvriront "
-"une porte qui garde un trésor antique...\n"
+"Les légendes parlent d'un piédestal sculpté dans de la pierre obsidienne et "
+"avec une mare de sang bouillonnant au sommet de sa surface incrustée d'os. "
```

Comment:
```suggestion
"avec une mare de sang bouillonnante au sommet de sa surface incrustée d'os. "
```
I just don't know here :D What is "bouillonnant" ? the "sang" or the "mare" ?

Reply:
Both could be used here, but "bouillonnant" is a bit easier on the tongue.